msc/views.py

```diff
@@ -341,4 +342,113 @@ def validUserRequest(request):
     # TODO
     return True
 
+@app.route('/new_customer_request', methods=['GET'])
+def new_customer_request():
+    return render_template('new_customer.html')
 
+def validCustomerAdd(request):
+    # TODO
+    return True
+
+@app.route('/create_customer_request', methods=['POST'])
+def create_customer_request():
+    if not session.get('logged_in'):
+        abort(401)
+
+    error = None
+    if not validCustomerAdd(request):
+        error = 'Invalid data entered'
+        print('invalid data entered')
+        return render_template('new_customer.html', error=error)
+
+    print('start add process...')
```

Comment:
Good rule, I'll do that going forward.
On Nov 6, 2015 20:23, "Dan Nascimbeni" <notifications@github.com> wrote:

> In msc/views.py
> <https://github.com/dnascimb/msc/pull/8#discussion_r44204633>:
>
> > +def validCustomerAdd(request):
> > +    # TODO
> > +    return True
> > +
> > +@app.route('/create_customer_request', methods=['POST'])
> > +def create_customer_request():
> > +    if not session.get('logged_in'):
> > +        abort(401)
> > +
> > +    error = None
> > +    if not validCustomerAdd(request):
> > +        error = 'Invalid data entered'
> > +        print('invalid data entered')
> > +        return render_template('new_customer.html', error=error)
> > +
> > +    print('start add process...')
>
> best to comment or remove print statements before commiting
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dnascimb/msc/pull/8/files#r44204633>.
>